### ISSUE TITLE
Facing the issue while run the neus file 

### ISSUE THREAD

**Issue by nikitelement** [liuyuan-pal/SyncDreamer#68]
` int D = 1; Derived = torch::nn::ConvTranspose1dImpl; std::ostream = std::basic_ostream<char>]’
/anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/modules/conv.h:282:8:   required from here
/anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/modules/conv.h:313:39: error: no matching function for call to ‘torch::nn::detail::ConvNdOptions<1>::padding() const’
  313 |     return std::get<ExpandingArray<D>>(this->options.padding());
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/data/dataloader_options.h:3,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/data/dataloader/base.h:3,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/data/dataloader/stateful.h:4,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/data/dataloader.h:3,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/data.h:3,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/all.h:9,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/extension.h:5,
                 from /mnt/batch/tasks/shared/LS_root/mounts/clusters/kpitgpu/code/Users/nikit/SyncDreamer/raymarching/src/bindings.cpp:1:
/anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/options/conv.h:61:3: note: candidate: ‘decltype (*(torch::nn::detail::ConvNdOptions<D>*)this) torch::nn::detail::ConvNdOptions<D>::padding(const int&) [with long unsigned int D = 1; decltype (*(torch::nn::detail::ConvNdOptions<D>*)this) = torch::nn::detail::ConvNdOptions<1>&]’
   61 |   TORCH_ARG(padding_t, padding) = 0;
      |   ^~~~~~~~~
/anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/options/conv.h:61:3: note:   candidate expects 1 argument, 0 provided
/anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/options/conv.h:61:3: note: candidate: ‘decltype (*(torch::nn::detail::ConvNdOptions<D>*)this) torch::nn::detail::ConvNdOptions<D>::padding(int&&) [with long unsigned int D = 1; decltype (*(torch::nn::detail::ConvNdOptions<D>*)this) = torch::nn::detail::ConvNdOptions<1>&]’
   61 |   TORCH_ARG(padding_t, padding) = 0;
      |   ^~~~~~~~~
/anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/options/conv.h:61:3: note:   candidate expects 1 argument, 0 provided
In file included from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/functional/conv.h:3,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/functional.h:4,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn.h:4,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/all.h:16,
                 from /anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/extension.h:5,
                 from /mnt/batch/tasks/shared/LS_root/mounts/clusters/kpitgpu/code/Users/nikit/SyncDreamer/raymarching/src/bindings.cpp:1:
/anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/options/conv.h:64:18: note: candidate: ‘decltype(auto) torch::nn::detail::ConvNdOptions<D>::padding(std::initializer_list<long int>) [with long unsigned int D = 1]’
   64 |   decltype(auto) padding(std::initializer_list<int64_t> il) {
      |                  ^~~~~~~
/anaconda/envs/syncdreamer/lib/python3.10/site-packages/torch/include/torch/csrc/api/include/torch/nn/options/conv.h:64:18: note:   candidate expects 1 argument, 0 provided
ninja: build stopped: subcommand failed.`

i have trained the model on your sample data but facing the issue while run this command
`python train_renderer.py -i output/aircraft/0.png -n aircraft-neus -b configs/neus.yaml -l output/renderer`

**Comment by zyshan0929** [comment]
I met the same issue on RTX3090
